app/team.go

```diff
@@ -1165,7 +1165,12 @@ func (a *App) RestoreTeam(teamId string) *model.AppError {
 }
 
 func (a *App) GetTeamStats(teamId string) (*model.TeamStats, *model.AppError) {
-	tchan := a.Srv.Store.Team().GetTotalMemberCount(teamId)
+	tchan := make(chan store.StoreResult, 1)
```

Comment:
Just curious, is this required to be async here?

Reply:
Yes, because we want to run multiple SQL queries concurrently